nextcloudappstore/core/templates/app/detail.html

```diff
@@ -110,5 +103,33 @@ <h4 id="downloads">{% trans "Downloads" %}</h4>
             </section>
         </div>
     </div>
+    {% if object.issue_tracker or object.discussion or request.user.is_authenticated %}
+        <div class="row app-comments">
+            <ul class="col-md-12">
+                <h4 class="section-heading" id="comments">{% trans 'Comments' %}</h4>
+                {% if object.issue_tracker %}
+                    <li><a target="_blank" rel="noreferrer noopener" href="{{ object.issue_tracker }}" class="btn btn-secondary">{% trans 'report problem' %}</a></li>
```

Comment:
I think these button texts should start with a capital letter like all the previous ones around the site. Or we can somehow rationalize the inconsistency.